zokrates_core_test/tests/tests/range_check/assert_lt_u8_constant.json

```diff
@@ -0,0 +1,67 @@
+{
+  "entry_point": "./tests/tests/range_check/assert_lt_u8_constant.zok",
```

Comment:
Should we have a test with a value which is outside the u8 range and check that it fails, or is it handled in some other test for usize inputs?